src/modules/auth/auth.controller.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  Body,
+  Controller,
+  Get,
+  HttpStatus,
+  Param,
+  Post,
+  Req,
+  UseGuards,
+} from '@nestjs/common';
+import { ApiOperation, ApiTags, ApiResponse } from '@nestjs/swagger';
+import { AuthGuard } from '@nestjs/passport';
+
+import { Doctor } from '@entities/Doctor';
+import { AuthService } from '@modules/auth.service';
+import { LoginDoctorDto } from '@modules/dto/login-doctor.dto';
+
+
+@ApiTags('auth')
+@Controller('auth')
+export class AuthController {
+  constructor(private readonly authService: AuthService) { }
+
+  @Post('login')
+  @ApiOperation({ summary: 'Doctor login' })
+  @ApiResponse({
+    status: 200,
+    description: 'Doctor was logged in',
+  })
+  @ApiResponse({
+    status: 401,
+    description: 'Invalid email or password',
+  })
+
+  async login(@Body() dto: LoginDoctorDto) {
```

Comment:
![Uploading Screenshot 2023-04-10 at 17.54.02.png…]()

Reply:
I don't see what the problem is, but I assume it's "try - catch" but we have a filter for that

Reply:
its not "try - catch", you have not follow checklist, you don't have types

Reply:
done